Resources/Prototypes/_White/Entities/Mobs/NPCs/headcrab.yml

```
@@ -0,0 +1,145 @@
+#    - Xeno
+#    - Hissing
+#    - Draconic
+#    - TauCetiBasic
+
+# HEADCRABS
+
+- type: entity
+  parent: BaseMobHeadcrab
+  id: MobHeadcrab
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)